tests/quantization/torchao_integration/test_torchao.py

```diff
@@ -349,7 +349,7 @@ def check_serialization_expected_output(self, device, expected_output):
         with tempfile.TemporaryDirectory() as tmpdirname:
             self.quantized_model.save_pretrained(tmpdirname, safe_serialization=False)
             loaded_quantized_model = AutoModelForCausalLM.from_pretrained(
-                self.model_name, torch_dtype=torch.bfloat16, device_map=device
+                tmpdirname, torch_dtype=torch.bfloat16, device_map=device
```

Comment:
Definitely make sense to me, but since @SunMarc is the author of this part of code, let's also get his response before we merge.